GoogleDataTransport/GDTCCTLibrary/GDTCCTUploader.m

```diff
@@ -175,14 +183,19 @@ - (nullable NSURL *)serverURLForTarget:(GDTCORTarget)target {
     case kGDTCORTargetCSH:
       return CSHServerURL;
 
+      // copybara:insert_begin(Reserve private endpoint)
+      //    case kGDTCORTargetINT:
```

Comment:
in the sample import, this case is indented 4 too many, so probably `// case kGDTCORTargetInt:`

Reply:
Fixed.